### PR TITLE
removed password from settings.py

### PR DESCRIPTION
**Pull request by anuj97**
@Samy-33 removed the password as you asked; resending the PR

### CODE REVIEW

**Comment by arshadsid** [comment]
@anuj97  flake8 and isort?

**Comment by Samy-33** [comment]
@anuj97 , please pull, resolve conflicts and push again.

File: FusionIIIT/Fusion/settings.py [hunk 1]
```diff
@@ -1,198 +0,0 @@
...
-USE_L10N = False
-
-USE_TZ = True
-
-SITE_ID = 1
-# Static files (CSS, JavaScript, Images)
-# https://docs.djangoproject.com/en/1.11/howto/static-files/
-
-STATIC_URL = '/static/'
-STATIC_ROOT = '/static'
```
Comment (Samy-33) [inline FusionIIIT/Fusion/settings.py]: You removed settings.py?

File: FusionIIIT/Fusion/urls.py [hunk 2]
```diff
@@ -22,5 +22,7 @@
     url(r'^login/', auth_views.login, name='login'),
     url(r'^logout/', auth_views.logout, name='logout'),
     url(r'^academic-procedures/', include('applications.academic_procedures.urls',
-                                          namespace='procedures')),
+        namespace='procedures')),
```
Comment (Samy-33) [inline FusionIIIT/Fusion/urls.py]: this was fine before.

**Comment by anuj97** [comment]
@Samy-33 done

File: FusionIIIT/Fusion/urls.py [hunk 3]
```diff
@@ -13,22 +13,14 @@
...
-    url(r'^accounts/', include('allauth.urls')),
-    url(r'^ocms/', include('applications.online_cms.urls')),
     url(r'^login/', auth_views.login, name='login'),
     url(r'^logout/', auth_views.logout, name='logout'),
     url(r'^academic-procedures/', include('applications.academic_procedures.urls',
                                           namespace='procedures')),
-
 ]
-if settings.DEBUG:
-    urlpatterns += static(settings.MEDIA_URL, document_root=settings.MEDIA_ROOT)
```
Comment (Samy-33) [inline FusionIIIT/Fusion/urls.py]: @anuj97 I see some undesired changes, in red. They were removed.
e72b5753-e7b7-4369-b160-d4cda6b0365c

File: FusionIIIT/Fusion/settings.py [hunk 4]
```diff
@@ -196,5 +196,3 @@
 
 STATIC_URL = '/static/'
 STATIC_ROOT = '/static'
-MEDIA_ROOT = os.path.join(BASE_DIR, 'media')
-MEDIA_URL = '/media/'
```
Comment (Samy-33) [inline FusionIIIT/Fusion/settings.py]: These shouldn’t be removed.

**Comment by Samy-33** [comment]
@anuj97 , travis fail

**Comment by anuj97** [comment]
just performed a rollback. gimme some time. i will be done soon.

**Comment by anuj97** [comment]
@Samy-33 please review

**Comment by Samy-33** [comment]
Just comment once you are done. Is it done?

**Comment by anuj97** [comment]
@Samy-33 yep, that's a green one

**Comment by Samy-33** [comment]
Ok, great.

**Pull request merged**